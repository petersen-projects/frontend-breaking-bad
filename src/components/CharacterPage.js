import React from 'react';

import CharacterList from './CharacterList';
import CharacterListFilter from './CharacterListFilter';
import Header from './Header';

export const CharacterPage = () => (
   <div className="character-page-container">
      <Header />
      <CharacterListFilter />
      <CharacterList />
   </div>
);

export default CharacterPage;