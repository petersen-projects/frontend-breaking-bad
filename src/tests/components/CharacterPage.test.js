import React from 'react';
import { shallow } from 'enzyme';
import CharacterPage from '../../components/CharacterPage';

test('should render CharacterPage correctly', () => {
   const wrapper = shallow(<CharacterPage />);
   expect(wrapper).toMatchSnapshot();
});
