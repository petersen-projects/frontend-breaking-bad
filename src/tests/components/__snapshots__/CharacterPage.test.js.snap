// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render CharacterPage correctly 1`] = `
<div
  className="character-page-container"
>
  <Header />
  <CharacterListFilter />
  <CharacterList />
</div>
`;
